addons/stock_account/models/stock_move.py

```diff
@@ -5,7 +5,7 @@
 
 from odoo import api, fields, models, _
 from odoo.exceptions import UserError
-from odoo.tools import float_compare, float_round, float_is_zero
```

Comment:
Maybe explain in the commit message why we earn performance on this + metrics if possible